README.md

```diff
@@ -41,7 +41,24 @@ These commands should resolve this issue.
 - Run `npm install` to install the dependencies so that your IDE can recognize them
 - Don't run `npm start` as the frontend is served in the frontend container
 
-- localhost - React application
+- localhost:3000 - React application
+
+## Production SSL Setup
+
+1. Ensure your server has a public IP address accessible from the internet
+2. Test the SSL certificate generation first with dry-run:
+   ```bash
+   docker compose run --rm certbot certonly --webroot --webroot-path /var/www/certbot/ --dry-run -d example.org
```

Comment:
please advice we manage ssl certificates or leave it to cloudflare?
If use cloudflare auto ssl certificates, I may need to remove those related codes